content/Solutions/Optimizations/Security.md

```diff
@@ -1,20 +1,36 @@
 ---
 title: "Security Recommendations"
-description: "Best practices when tuning TrueNAS to minimize security vulnerabilities."
+description: "Best practices to minimize security vulnerabilities in TrueNAS."
 weight: 30
 aliases:
   - /core/solutions/optimizations/security/
 tags:
  - coressh
  - scalessh
+keywords:
+ - TrueNAS Security
 ---
 
 {{< toc >}}
 
-When using services on TrueNAS, especially services that allow outside connections, there are some best practices to follow to ensure your system is safe and secure.
-Several different system services are disscused in this article.
+Follow these best practices to minimize security vulnerabilities in TrueNAS.
 
-## iSCSI
+## General Recommendations
+
+1. Specially modifying the base TrueNAS firmware image is unsupported and can create security issues.
+2. Keep TrueNAS up to date with the most recent maintenance releases of the installed major version.
```

Comment:
"Keep TrueNAS up to date with the most recently updates for your supported version"